11-apply-manifest-code/README.md

```diff
@@ -247,3 +247,12 @@ hello! from 127.0.0.1
 # Next steps
 
 Now that you have learned about applying existing module content you can harness the power of the Puppet forge to manage infrastructure and deploy great applications!
+
+Congratulations! You should now have a basic understanding of `bolt` and Puppet Tasks. Here are a few ideas for what to do next:
+
+* Explore content on the [Puppet Tasks Playground](https://github.com/puppetlabs/tasks-playground)
+* Get reusable tasks and plans from the [Task Modules Repo](https://github.com/puppetlabs/task-modules)
+* Search Puppet Forge for [Tasks](https://forge.puppet.com/modules?with_tasks=yes)
+* Start writing Tasks for one of your existing Puppet modules
+* Head over to the [Puppet Slack](https://slack.puppet.com/) and talk to the `bolt` developers and other users
+* Try out the [Puppet Development Kit](https://puppet.com/download-puppet-development-kit) [(docs)](https://docs.puppet.com/pdk/latest/index.html) which has a few features to make authoring tasks even easier
```

Comment:
Hm, it feels like this should be boilerplate at the top level, rather than moving it onto the end of the last  task

Reply:
maybe, I was trying not to make too large a change since I don't work on the team and don't know what their vision and such for this is.